Modules/Module_Processing/Module_Population/xASL_im_CreateGroupAnalysisMask.m

```diff
@@ -241,25 +232,6 @@
 x.S.CorSlices = [];

```

Comment:
We have a separate visualization init function where these stuff should live. Not important for now, just that you know: `xASL_init_VisualizationSettings`

Reply:
OK. Good to know. I wasn't really checking the old code except for removing comments that are there for ages :)